.github/workflows/release-helm-chart.yaml

```diff
@@ -0,0 +1,68 @@
+name: release-helm-chart
+on:
+# TODO: The following commented lines should be used depending on the release strategy
+#  release:
+#    types:
+#      - released
+# OR for a full release workflow
+#  push:
+#    tags:
+#        - 'v*.*.*'
+  workflow_dispatch:
+    inputs:
+      operatorVersion:
+          description: Operator bundle version
+          default: 0.0.0
+          type: string
+      limitadorVersion:
+        description: Limitador version
+        default: latest
+        type: string
+      releaseId:
+        description: Release ID
+        default: 0
+        type: string
+
+jobs:
+  chart_release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.ref }}
+        fetch-depth: 0
+
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+    - name: Build the Helm Chart manifests
+      run: |
+        make helm-build \
```

Comment:
I feel like we should not be doing this. The git commit (or tag) we are releasing should have done previously `make helm-build`. Any thought?

Reply:
Yes, I was thinking it should be part of the building manifests steps just before tagging... However, just a reminder that currently we do that manually before tagging and then let the CI to build the manifests again and publish them to Quay... meaning it's kind of the same in this case, we could run this target and commit it with the rest of the manifests when releasing... but having a separated workflow is convenient when you want to attach charts to other already released operators. What do you reckon?

Reply:
> currently we do that manually before tagging and then let the CI to build the manifests again and publish them  to Quay.

Isn't that redundant? What I would do, maybe, it is a test to run (on release event) `make helm-build` and check that no files changed. Same we do with `go generate` or `make bundle`.

> having a separated workflow is convenient when you want to attach charts to other already released operators

Not sure I follow. 

Anyway, I was not asking for changes actually. Whatever you decide is ok for me on this regard.

Reply:
I agree with you, we'll need to address the redundancy in our workflows, part should be reflected in https://github.com/Kuadrant/architecture/pull/41